### PR TITLE
CP-34000: decimal network and disk access rate unit calculations

### PR DESCRIPTION
**Pull request by AaronRobson**
Though this was shown in units like kB/s and MB/s (decimal units) it was calculated in
binary terms i.e. 1MB/s was treated as 1024kB/s instead of the expected 1000kB/s.

Updated translations for the units to be of the form 'MB/s' instead of 'MBps'.

Signed-off-by: Aaron Robson <aaron.robson@citrix.com>

### CODE REVIEW

File: XenModel/Messages.ja.resx [hunk 1]
```diff
@@ -13161,7 +13161,7 @@ SR-IOV ネットワークを有効にするには、サーバーの再起動が
     <value>kW</value>
   </data>
   <data name="VAL_KILRATE" xml:space="preserve">
-    <value>Kbps</value>
+    <value>KB/s</value>
```
Comment (kc284) [inline XenModel/Messages.ja.resx:13164]: Lowercase 'k'.

File: XenModel/Messages.zh-CN.resx [hunk 2]
```diff
@@ -13158,7 +13158,7 @@ VM 克隆使用文件管理器的快照和克隆功能来实现高性能，并
     <value>kW</value>
   </data>
   <data name="VAL_KILRATE" xml:space="preserve">
-    <value>KBps</value>
+    <value>KB/s</value>
```
Comment (kc284) [inline XenModel/Messages.zh-CN.resx:13161]: Lowercase 'k'.

File: XenModel/Messages.ja.resx (hunk 1 above)
Comment (AaronRobson) [reply XenModel/Messages.ja.resx]: Good spot, corrected.

File: XenModel/Messages.zh-CN.resx (hunk 2 above)
Comment (AaronRobson) [reply XenModel/Messages.zh-CN.resx]: Also corrected.

**Pull request merged**